rust/relay/server/src/server.rs

```diff
@@ -729,13 +729,6 @@ where
                 (channel.allocation, channel.peer_address),
                 (sender, requested_channel),
             );
-            self.pending_commands
-                .push_back(Command::CreateChannelBinding {
-                    client: sender,
-                    channel_number: requested_channel,
-                    peer: peer_address,
-                    allocation_port: channel.allocation,
-                });
```

Comment:
This actually discovered another bug where we only added the channel binding for eBPF once it is was refreshed for the first time (so only after 5min of using the channel).